homeassistant/helpers/selector.py

```diff
@@ -448,11 +463,20 @@ def __init__(self, config: ColorTempSelectorConfig | None = None) -> None:
 
     def __call__(self, data: Any) -> int:
         """Validate the passed selection."""
+        range_min = self.config.get("min")
+        range_max = self.config.get("max")
+
+        if not range_min:
+            range_min = self.config.get("min_mireds")
+
+        if not range_max:
+            range_max = self.config.get("max_mireds")
```

Comment:
Note: the code as it currently stands sets the min and max values to the same for both the mireds and kelvin selectors (153 to 500), which doesn't seem right?  These values seem to work well for mireds, but for kelvin the light doesn't change temperature much.

Guessing this needs different values for min and max if we're using the kelvin selector, though not sure what values should be set.

![chrome_VjCP8rzWDh](https://github.com/home-assistant/core/assets/4665754/17a6196a-1b34-4d89-8e14-4be3fdd52f9e)

Reply:
The [frontend merge request](https://github.com/home-assistant/frontend/pull/18627/files) is also needed. 🙂